mongodb_Python/utilities/base_mongo.py

```diff
@@ -0,0 +1,76 @@
+from utilities.configuration import Configuration
+import pymongo
+
+
+class BaseMongo:
+    def __init__(self):
+        self.client = pymongo.MongoClient(Configuration().connection_URL)
+        self.db = self.client[Configuration().db_name]
+        # self.collection_name = self.client[collection_name]
+
+    def add_new_data(self, collection_name: str, document_data: dict | list):
+        """
+        :param collection_name: name of collection
+        :param document_data: the data of new document(s), will add one document if document_data get dict
+               or many documents if document_data get list of dicts
+        :return:
+        """
+        if type(document_data) is dict:
+            self.db[collection_name].insert_one(document_data)
+        elif type(document_data) is list:
+            self.db[collection_name].insert_many(document_data)
+
+    def find_document(self, collection_name: str, search_options: dict = {}, out_put_options: dict = {}):
+        """
+        :param collection_name: name of collection
+        :param search_options: optional argument
+        :param out_put_options: optional argument
+        :return: result of search dict
+        """
+        search_result = self.db[collection_name].find_one(search_options, out_put_options)
+        return search_result
+
+    def find_documents(self, collection_name: str, search_options: dict = {}, out_put_options: dict = {}):
```

Comment:
it is not a good practice to have a mutable type for the default parameter. But let it be in such case